pkg/virt-launcher/virtwrap/device/hostdevice/sriov/hostdev_test.go

```diff
@@ -77,6 +77,132 @@ var _ = Describe("SRIOV HostDevice", func() {
 			Expect(err).To(HaveOccurred())
 		})
 
+		It("creates a device with a custom guest PCI address", func() {
```

Comment:
Any reason you placed these tests in the middle here? Before and after we have failing scenarios.

Also, how is this one different from the [existing](https://github.com/kubevirt/kubevirt/pull/7422/commits/c9cf19b8a84a704b6dce1a1aa1e730bc0a51d74c#diff-1b1b36aee3a76abe40a07b703d80cc1d14024be09e84400a2ec29005d68d11e1R274) `"creates 1 device that includes guest PCI addresses"`?

Reply:
> Also, how is this one different from the existing "creates 1 device that includes guest PCI addresses"?
Thanks for catching this - it will be removed.